.travis.yml

```diff
@@ -5,16 +5,29 @@ matrix:
   include:
   - python: 3.8
     dist: xenial
-    env: TF=0
```

Comment:
The changes here are because I wanted to have a test where neither TF or Torch was installed.

Reply:
I like this change, makes it easier to see which jobs use torch (it was too implicit before)